py_experimenter/experimenter.py

```diff
@@ -203,14 +328,35 @@ def execution_wrapper(self, approach, custom_fields: dict, keyfields, result_pro
             else:
                 result_processor._change_status('done')
 
-    def reset_experiments(self, status= 'error') -> None:        
-        keyfields,entries = self._dbconnector.delete_experiments_with_status(status)
+    def reset_experiments(self, status= 'error') -> None:
+        """
+        Deletes the experiments of the database table having the given status and adds the according keyfields as new rows.
```

Comment:
Deletes the experiments of the database table having the given status and adds the according keyfields as new rows. —>  Deletes the experiments of the database table having the given status. Afterwards all rows that have been deleted from the database are added to the table again.

(Finde es komisch da von den keyfields zu sprechen)

Reply:
Afterwards all rows that have been deleted from the database are added to the table again. -> Afterwards all rows that have been deleted from the database are added to the table again featuring "created" as a status.